lxd/networks.go

```diff
@@ -510,6 +510,19 @@ func (n *network) Start() error {
 		}
 	}
 
+	// IPv6 bridge configuration
+	if !shared.StringInSlice(n.config["ipv6.address"], []string{"", "none"}) {
+		err := ioutil.WriteFile(fmt.Sprintf("/proc/sys/net/ipv6/conf/%s/autoconf", n.name), []byte("0"), 0)
+		if err != nil {
+			return err
+		}
+
+		err = ioutil.WriteFile(fmt.Sprintf("/proc/sys/net/ipv6/conf/%s/accept_dad", n.name), []byte("0"), 0)
```

Comment:
We're a router on a bridge, we don't want containers on that bridge to be able to remove our IP by conflicting with it. Also, DAD adds a 5s delay in setting an IPv6 address which slows down everything.